crates/mysten-metrics/src/lib.rs

```diff
@@ -630,6 +635,20 @@ pub async fn metrics(
     }
 }
 
+#[macro_export]
+macro_rules! system_invariant_violation {
+    ($msg:expr) => {
+        if cfg!(debug_assertions) {
```

Comment:
Maybe `debug_fatal`?

Reply:
TIL! I will see if I could just modify that instead.